### ISSUE TITLE
A: Prepare Digital City Map Files for Bytes/Open Data

### ISSUE THREAD

**Issue by aferrar** [NYCPlanning/edm-overview#111]
**12/5/19: All necessary info received from TRD. Finishing script for distribution**

Also added new data sets with schema breakdown given to us from TRD. Listed below:

* DCM
  * Feature Class - DCM
  * Retain Fields - **Type**, **Feature_type**, **Boro_nm**, **Jurisidication** (used for determining pierhead and bulkhead line type symbology based on the agency which the jurisdiction falls under), **Record_st**, **Edit_DT**

* DCM_SCL
  * Feature Class - DCM_SCL
  * Retain Fields - **Old_ROW_Name**, **Route_type** (Streets app will filter out "Fromer_ROW" values), **Streetwidth**, **ROW_Name**, **HonoraryName**, **Roadway_type**, **Boro_name**, **Feature_status**, **DCM_Type**, **Build_status**, **Marginal_wharf_place**, **Step_stair_St** (Streets app will use for toggle layer), **CCO**, **Edit_DT**, **Paper_St** (Streets app will use for toggle layer)

* DCM_Arterial_Major_Streets
  * Feature Class - DCM_Arterials_Mjr_Sts (Streets app will use for toggle layer)
  * Retain Fields - All fields

* DCM_CityMapChanges (toggle layer)
  * Feature Class - DCM_nymi
  * Retain Fields - All fields

* DCM_StreetNameChanges_Points (toggle layer)
  * Feature Class - DCP_TRD_StreetNameChanges_Points
  * Retain Fields - All fields

* DCM_StreetNameChanges_Lines (toggle layer)
  * Feature Class - DCM_NmChng_Sts
  * Retain Fields - All fields

* DCM_StreetNameChanges_Areas (toggle layer)
  * Feature Class - DCP_TRD_StreetName_Changes_Areas
  * Retain Fields - All fields

* DCM_FinalSection_Index
  * Feature Class - DCM_FS_Index
  * Retain Fields - All fields (confirmed with TRD 12/3/19)

* DCM_Alteration_Maps
  * Feature Class - DCM_Alt_maps_Phase2
  * Retain Fields - **ALTMAPPDF**, **Complt_status**, **Inset**

**12/3/19: Received updated field details from Andrew in TRD**

**11/25/19: Andrew (TRD) sent the file names and field notes**

**10/22/19: TRD expects to send updated file at the end of October (work for Sprint W or X)**

Met w/ Labs and TRD on Wed, 7/3: agreed to postpone the posting to Bytes/Open Data for another 2 weeks so TRD can apply schema changes.

Will check back w/ TRD week of 7/15 to see if files are ready. Need to prepare webteam and Open Data for new web pages.

Script necessary for pulling DCM Feature Classes from sde@GISPROD and distributing to appropriate locations:
- [x] Shapefiles in M:\GIS\BytesProduction\ (remove all fields except the following)
    - DCM: Boro_nm, Feature_type, type, Source, Jurisdiction, Ownership_status
    - DCM_SCL: Streetwidth, Official_street_name, Roadway_type, Boro_name, feature_status
    - DCM_NYMI: app_num, Status, Effective, borocode, altmappdf, chg_type, scanned, project_name, filed_status
- [x] Layers and xmls in M:\GIS\DATA\City Map (waiting on DCM schema changes from TRD: approximated to take 2 weeks on 07/02/2019)

**Comment by croswell81** [comment]
@aferrar  please review the shapefile steps to reflect latest work.

**Comment by aferrar** [comment]
**12/13/2019**

Script and outputs are complete. Waiting for review from @croswell81 and Andrew English from TRD. Reached out to TRD on 12/10/2019.
